contracts/Voter.sol

```diff
@@ -12,16 +12,14 @@ import "./interfaces/IRomulusDelegate.sol";
 contract Voter is Ownable {
```

Comment:
Let's also add a `delegate` function to delegate votes to anyone from this `Voter`. See how line 26 in this file currently works